### PR TITLE
Parameter response tab in RFT plotter

### PR DESCRIPTION
**Pull request by lindjoha**
Included a new tab with plots for analysing the RFT response to parameters:

* bar chart with correlations. Option to select RFT vs parameters or parameter vs RFTs.
* scatter plot with RFT vs parameter. Vertical lines showing the observation with errors
* formations plot colored by parameter.

The functionality is inspired by the Parameter Analysis plugin and some of the functionality is generalized. Some of the figures (bar chart and scatter plot) are prepared for further generalization between the two. 

The parameter response tab will not appear when working with aggregated data. Or when for some reason no non-constant parameters are found.

### CODE REVIEW

File: webviz_subsurface/_utils/colors.py [hunk 1]
```diff
@@ -4,9 +4,7 @@
 
 def hex_to_rgb(hex_string: str) -> Tuple[float, float, float]:
     """Converts the given hex color to rgb tuple with floating point byte color values.
-
     Byte color channels: 0-255
-
```
Comment (anders-kiaer) [inline webviz_subsurface/_utils/colors.py:9]: Are these removed empty lines intentional?

File: webviz_subsurface/_utils/colors.py [hunk 2]
```diff
@@ -63,29 +55,50 @@ def hex_to_rgba_str(hex_string: str, opacity: float = 1.0) -> str:
...
 
+def rgba_to_str(rgba: Tuple[float, float, float, float]) -> str:
+    """Convert rgba tuple with floating point byte color values to string
+    `Input:`
+    * rgb - Tuple[float,float,float,float] - RGBA color on tuple format
+    `Return:`
+    RGBA color on string format "rgba(r,g,b,a)" where, channels r, g and b are
+    represented with byte color integer value 0-255, and a is represented by a
+    decimal number 0-1"""
+    return f"rgba({round(rgba[0])}, {round(rgba[1])}, {round(rgba[2])}, {rgba[3]})"
```
Comment (anders-kiaer) [inline webviz_subsurface/_utils/colors.py:76]: Feels a bit strange to have both `rgba_to_str` and `rgb_to_str`, but there are also other (existing) functions in `_utils/colors` which might be condensed perhaps. We leave it as is, but maybe we should make an issue on reducing number of color-functions if natural.

**Review by anders-kiaer — COMMENTED** [review]
Nice PR @lindjoha! 🚀 Worth a changelog entry?

File: webviz_subsurface/_figures/barchart.py [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+from .._utils.colors import hex_to_rgba
+
+
+class BarChart:
+    def __init__(self, series: pd.Series, n_rows: int, title: str, orientation: str):
+        self.series = series.tail(n=n_rows)
+        self.title = title
+        self.orientation = orientation
+        self._data = self.initial_data
```
[the remark this answers is not in the document]
Comment (lindjoha) [reply webviz_subsurface/_figures/barchart.py:13]: It is a function below with a `property `decorator. The decorator might be unnecessary. This whole class is more or less copied from the `correlation_figure.py` file in the `ParameterAnalysis `plugin. The idea was to replace it with this general class in a separate PR.

[the remark this answers is not in the document]
Comment (anders-kiaer) [reply webviz_subsurface/_figures/barchart.py:22]: Ah, ok, then we leave at is for now 🙂

**Review by anders-kiaer — APPROVED** [review]
LGTM after the modifications 👍

**Pull request merged**